### ISSUE TITLE
Pre-training?

### ISSUE THREAD

**Issue by llu025** [rulixiang/SCCNet#1]
Hello,

First of all, thank you for sharing your implementation of this method.
I noticed that in your implementation there is no pre-training, which is suggested by the authors to avoid trivial solutions. Also, have you tried with more difficult data sets (opt. vs sar) instead of using the (opt. vs opt.) data set you provide?

**Comment by rulixiang** [comment]
Yes, I didn't use pre-training in this code. Actually, the result obtained by this code is very different from  the groundtruth. I'm still not very sure if the alternate optimization implemented in this code is right. I haven't tried any other data sets so far.

**Comment by rulixiang** [comment]
> Hello,
> 
> First of all, thank you for sharing your implementation of this method.
> I noticed that in your implementation there is no pre-training, which is suggested by the authors to avoid trivial solutions. Also, have you tried with more difficult data sets (opt. vs sar) instead of using the (opt. vs opt.) data set you provide?

Yes, I didn't use pre-training in this code. Actually, the result obtained by this code is very different from the groundtruth. I'm still not very sure if the alternate optimization implemented in this code is right. I haven't tried other data sets so far.
